### PR TITLE
Add Seznam.cz

### PR DESCRIPTION
**Pull request by radeksimko**
Replaces #2694

### CODE REVIEW

**Review by kmpoppe — CHANGES_REQUESTED** [review]
Hello Radek!

Thank you for your contribution to this project! Could you please use tinypng.com to reduce the size of the logo? Even though it's only 2.19kb, tinyjpg reduces the size by 60%.

Thank you!

Kai

**Comment by radeksimko** [comment]
Of course, I am happy to.

Related question - would you like me to add this tip to the [contributing guidelines](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines) (next to OptiPNG)?

**Comment by radeksimko** [comment]
@kmpoppe PTAL.

**Comment by kmpoppe** [comment]
@radeksimko Will do later, thanks!

File: _data/email.yml [hunk 1]
```diff
@@ -157,6 +157,13 @@ websites:
       software: Yes
       doc: https://help.runbox.com/account-security/
 
+    - name: Seznam.cz
+      url: https://seznam.cz
+      twitter: seznam_cz
+      facebook: sledujseznam
+      img: seznamcz.png
+      tfa: No
+
```
Comment (kmpoppe) [inline _data/email.yml]: Please add `lang`-field.

Comment (kmpoppe) [reply _data/email.yml]:
```suggestion
      lang: cz
      
```

**Review by kmpoppe — CHANGES_REQUESTED** [review]

**Review by jtagcat — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**